accepted/00000-coco-attestation.md

```diff
@@ -0,0 +1,305 @@
+- Feature Name: Confidential Compute Attestation Report
+- Start Date: 2024-01-16
+
+# Summary
+[summary]: #summary
+
+SUSE Manager / Uyuni should help with Confidential Compute Attestation and being able to generate a report page for systems which should run in this mode.
+
+# Motivation
+[motivation]: #motivation
+
+More and more systems run as virtual machines or (public) cloud instances.
+Customers want to be sure that data of these VMs cannot be read by administrators or the cloud provider.
+Confidential Compute is the way to solve this problem.
+But that a VM run in confidential mode needs to be attested and checked on regular base.
+
+SUSE Manager should provide an attestation report for a system.
+Also a history of the past checks should be kept and shown.
+
+The attestation is a process of multiple checks which can also depend on the hardware and used environment.
+SUSE Manager should provide a framework for executing such checks, store the results in the database and show the latest and historic results.
+The number of checks should be easy extendable in future versions.
+
+
+# Detailed design
+[design]: #detailed-design
+
+Confidential Compute Attestation cannot be performed on all systems.
+It depends on the used hardware and environment where the instance is running on.
+In most cases it is not possible to detect if an attestation can be performed or not.
+
+This requires a per system configuration which should contain at least two configuration options:
+
+- Enable/Disable Confidential Compute Attestation (Checkbox)
+- Environment selection
+  - KVM Guest Attestation on AMD EPYC Milan CPUs
+  - KVM Guest Attestation on AMD EPYC Genoa CPUs
+  - Azure Attestation
+  - etc.
```

Comment:
Should this depend on the previous configuration option and only comes into affect if attestation could be performed ? And will the appropriated selection be done automatically by SUSE Manager based on some criteria or will this be left to the user to select the correct Environment?

Reply:
The Enable/Disable checkbox would show/hide this option.
You can only select an environment, when you want to perform an attestation.

As we cannot find out if this can be successful or not, the admin tell us what environment it is.